.github/workflows/test-native-gradle-plugin.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gradle-version: ["current", "7.3.3", "7.2", "7.1", "6.8.3","6.7.1"]
```

Comment:
I wanted to reduce consumption of hosted runners, since there is a limited number of them available for the entire organization at any given time, and 6 of them would be used for each push on each PR just for Gradle testing (and those are long running tests that keep runners busy for a long time - thus preventing others from testing their stuff).

What I believe to be the optimal CI setup:
* On a push to PR we only test first and last supported Gradle version, but configure matrix so that each runner runs only a certain part of tests. That way we get a much quicker response time.
* On a push to master we test more supported versions but configure it so that maybe more of them get tested on few runners (perhaps one or two that would be active for a long time).

I agree that this change shouldn't be here, so I'll revert it and open issue for further discussion.